src/react-native.ts

```diff
@@ -109,6 +135,19 @@ class Synchronizer {
       totalZatoshi: '0'
     }
   }
+
+  async sendToAddress(spendInfo: SpendInfo): Promise<PendingTransaction> {
+    const result = await RNZcash.spendToAddress(
+      spendInfo.zatoshi,
+      spendInfo.toAddress,
+      spendInfo.memo,
+      spendInfo.fromAccountIndex,
+      // TODO: ask is it okay to send this across the boundary or should it live on the native side and never leave?
```

Comment:
Seems ok to have fromAccountIndex as part of the API. A client may choose to have multiple accounts and would need to specify one in order to spend.

Reply:
If that's in response to my `TODO` question, I was trying to ask about the line below, rather than above the comment. In other words, is it okay to pass the spending key through like this? Or should we expect it to remain on the Android side, instead, for security?

Reply:
Yes, it is fine to pass the spending key. Since this is a library for a react native app, we can assume that the JavaScript side has access to the keys